client/app/queue/OrganizationUsers.jsx

```diff
@@ -111,6 +111,7 @@ export default class OrganizationUsers extends React.PureComponent {
       this.setState({
         organizationUsers: [...this.state.organizationUsers, response.body.users.data[0]],
         remainingUsers: this.state.remainingUsers.filter((user) => user.id !== value.id),
+        membershipRequests: this.state.membershipRequests.filter((mr) => parseInt(mr.userId) !== parseInt(value.id)),
```

Comment:
Linting error fix. parseInt needs a base. I'm not sure it's necessary to cast it as an integer however since it should be the same value in both instances.

```suggestion
        membershipRequests: this.state.membershipRequests.filter((mr) => parseInt(mr.userId, 10) !== parseInt(value.id, 10)),
```